Examples/MAX32690/Bootloader/bootloader.ld

```diff
@@ -32,7 +32,7 @@
  ******************************************************************************/
 
 BOOTLOADER_ORIGIN = 0x10000000;
-BOOTLOADER_LEN = 0x4000;
+BOOTLOADER_LEN = 0xA000;
```

Comment:
Need to be double check after apply below change
https://github.com/Analog-Devices-MSDK/msdk/pull/663#discussion_r1267154096

Reply:
It fits inside 0x4000 after applying above change.